cfme/tests/physical_infrastructure/test_physical_rack.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="function")]
+
+
+@pytest.fixture(scope="module")
+def physical_rack_collection(appliance, provider, setup_provider_modscope):
+    # Get and return the physical rack collection
+    return appliance.collections.physical_racks
+
+
+def test_physical_racks_view_displayed(physical_rack_collection, provider):
+    physical_racks_view = navigate_to(physical_rack_collection, 'All')
```

Comment:
I don't think this action warrants its own test. Any test using the racks collection will complete this action.

The following test is checking for refresh action in configuration dropdown, but doesn't actually take any action.

How about a single test that actually clicks on the refresh action in configuration dropdown? This will cover navigation, and iteration with the toolbar widget.

Is refresh the only option available in the configuration dropdown?

Reply:
Yes, the refresh is the only dropdown option. merged these two methods